### PR TITLE
DM-20972: Update miniconda to 4.7.10

### PR DESCRIPTION
**Pull request by gcomoretto**
These changes should be sufficient to support @brianv0 new environment.
I have also added a couple of improvements in the messaging.

### CODE REVIEW

File: bin/deploy [hunk 1]
```diff
@@ -455,7 +455,9 @@ main() {
     )
   fi
 
-  echo "Done. Run the following:"
+  echo ""
+  echo "::: Deploy completed."
+  echo "::: To active the just deployed environment use the following command:"
```
Comment (timj) [inline bin/deploy]: "To activate..."

File: bin/deploy [hunk 2]
```diff
@@ -455,7 +455,9 @@ main() {
     )
   fi
 
-  echo "Done. Run the following:"
+  echo ""
+  echo "::: Deploy completed."
```
Comment (timj) [inline bin/deploy]: I don't really understand what the `::: ` is for.

File: bin/deploy [hunk 3]
```diff
@@ -466,17 +468,16 @@ main() {
     elif [[ $SHELL =~ zsh$ ]]; then
       echo "    . ${LSSTSW}/bin/setup.sh"
     else
-      echo "    The setup script in ${LSSTSW}/bin most appropriate for ${SHELL}"
-      echo "    (Warning: ${SHELL} is an unsupported shell)"
+      echo "::: The setup script in ${LSSTSW}/bin most appropriate for ${SHELL}"
```
Comment (timj) [inline bin/deploy]: why the change from spaces to colons?

File: bin/envconfig [hunk 4]
```diff
@@ -114,7 +114,11 @@ if $INTERACTIVE ; then
     envs[l]=$(echo "${i}" | cut -f 1 -d ' ')
     echo "${l} - ${envs[${l}]}"
   done
+  echo "0 EXIT (return)"
   read -r -p "Choose environment [1 to ${l}]: " eid
```
Comment (timj) [inline bin/envconfig]: Maybe include the `0 to exit` in the prompt?

File: bin/deploy (hunk 2 above)
Comment (gcomoretto) [reply bin/deploy]: I tried to follow what was done above, but probably I should have not changed other's print outs.

File: bin/deploy (hunk 3 above)
Comment (gcomoretto) [reply bin/deploy]: same as above. I Put back the spaces.

File: bin/envconfig (hunk 4 above)
Comment (gcomoretto) [reply bin/envconfig]: 0 is not an environment. But if you feel it is better to include it, I do.

Comment (timj) [reply bin/envconfig]: I mean a prompt something like:
```
Choose environment [1 to ${l}; 0 to exit]: 
```
